### PR TITLE
ascanrules: Add policy alert tags

### PR DESCRIPTION
**Pull request by kingthorin**
## Overview
- CHANGELOG > Add note and fix note.
- Scan rules > Add PolicyTag values.
- Unit tests > Assert the new tags.
- Build file > Update the required version of commonlib.

## Related Issues
- zaproxy/zap-extensions#5930

## Checklist
- [na] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [x] Write tests
- [x] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

### CODE REVIEW

File: addOns/ascanrules/ascanrules.gradle.kts [hunk 1]
```diff
@@ -13,7 +13,7 @@ zapAddOn {
         dependencies {
             addOns {
                 register("commonlib") {
-                    version.set(">= 1.21.0 & < 2.0.0")
+                    version.set(">= 1.29.0 & < 2.0.0")
```
Comment (thc202) [inline addOns/ascanrules/ascanrules.gradle.kts:16]: Should be mentioned in the changelog.

File: addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java [hunk 2]
```diff
@@ -54,11 +57,21 @@ public class PathTraversalScanRule extends AbstractAppParamPlugin
...
+    private static final Map<String, String> ALERT_TAGS;
+
+    static {
+        Map<String, String> alertTags =
+                new HashMap<>(
+                        CommonAlertTag.toMap(
+                                CommonAlertTag.OWASP_2021_A01_BROKEN_AC,
+                                CommonAlertTag.OWASP_2017_A05_BROKEN_AC,
+                                CommonAlertTag.WSTG_V42_ATHZ_01_DIR_TRAVERSAL));
+        alertTags.put(PolicyTag.DEV_FULL.getTag(), "");
```
Comment (thc202) [inline addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java:69]: Missing `DEV_STD`.

File: addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SstiBlindScanRule.java [hunk 3]
```diff
@@ -50,11 +53,19 @@ public class SstiBlindScanRule extends AbstractAppParamPlugin implements CommonA
...
+    private static final Map<String, String> ALERT_TAGS;
+
+    static {
+        Map<String, String> alertTags =
+                new HashMap<>(
+                        CommonAlertTag.toMap(
+                                CommonAlertTag.OWASP_2021_A03_INJECTION,
+                                CommonAlertTag.OWASP_2017_A01_INJECTION,
+                                CommonAlertTag.WSTG_V42_INPV_18_SSTI));
+        alertTags.put(PolicyTag.DEV_FULL.getTag(), "");
```
Comment (thc202) [inline addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SstiBlindScanRule.java:65]: Missing sequence and API.

**Comment by kingthorin** [comment]
Fixed

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**